File: 94-binary-tree-inorder-traversal/94-binary-tree-inorder-traversal.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right
class Solution:
    def inorderTraversal(self, root: Optional[TreeNode]) -> List[int]:
        res = []   
        def bst(val):
            if val:
                bst(val.left)
                res.append(val.val)
                bst(val.right)
                
        
        bst(root)
        return res
            
        

File: 94-binary-tree-inorder-traversal/94-binary-tree-inorder-traversal.cs
/**
 * Definition for a binary tree node.
 * public class TreeNode {
 *     public int val;
 *     public TreeNode left;
 *     public TreeNode right;
 *     public TreeNode(int val=0, TreeNode left=null, TreeNode right=null) {
 *         this.val = val;
 *         this.left = left;
 *         this.right = right;
 *     }
 * }
 */
public class Solution {
    public IList<int> InorderTraversal(TreeNode root) {
        List<int> list = new List<int>();
        
        void inOrder(TreeNode root){
            if (root !=null){
                inOrder(root.left);
                list.Add(root.val);
                inOrder(root.right);

                
            }
        }
        
        inOrder(root);
        return list;
        
    }
}